src/Frame.js

```diff
@@ -0,0 +1,39 @@
+class Frame {
+  constructor() {
+    this.currentroll = 1;
+    this.rollOne = 0;
+    this.rollTwo = 0;
+    this.endFrame = false;
+    this.frameType = 'Normal';
```

Comment:
I think there is too much duplicated information here again.
You only really need the rolls, and maybe the currentroll (although you could get away with that too, by having the rolls initialised at null or undefined). Everything else can be calculated based on that information.